mlflow/server/js/src/common/utils/FeatureUtils.ts

```diff
@@ -9,26 +9,27 @@ export const shouldDisableLegacyRunCompareCharts = () => false;
 /**
  * UI feature preview: displays data lineage (datasets used) in experiment runs
  */
-export const shouldEnableExperimentDatasetTracking = () => true;
+export const shouldEnableExperimentDatasetTracking = () => false;
 /**
  * UI feature preview: enables artifact-based ML experiment output data analysis, used for evaluating LLM prediction data
  */
 export const shouldEnableArtifactBasedEvaluation = () => true;
 
-export const shouldEnableDatasetsDropdown = () => false;
-
 export const shouldEnableDeepLearningUI = () => false;
 
+export const shouldEnableDeepLearningUIPhase2 = () => false;
```

Comment:
Are we going to set this to true?

Reply:
It looks like the OSS backend is not ready for this.

Reply:
@harupy correct, we wait for https://github.com/mlflow/mlflow/issues/10455